Bottom: 3fe1a430b28d9deb750fd2bd4571ad4a55532d72
Top:    b153c46f928ef9901a988e8b47044b7cf3b3ebc5
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-18 02:11:56 +0000

Refresh of remove-calls-to-savefindflag

---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index bcbd264..07fdac7 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -535,7 +535,6 @@ netlist_select_cb (GtkWidget * widget, gpointer data)
   if (selected_net == node_selected_net)
     node_selected_net = NULL;
 
-  SaveFindFlag (FOUNDFLAG);
   InitConnectionLookup ();
   ResetConnections (true, FOUNDFLAG);
 
@@ -546,7 +545,6 @@ netlist_select_cb (GtkWidget * widget, gpointer data)
   SelectConnection (select_flag);
   ResetConnections (false, FOUNDFLAG);
   FreeConnectionLookupMemory ();
-  RestoreFindFlag ();
   IncrementUndoSerialNumber ();
   Draw ();
 }
diff --git a/src/hid/lesstif/netlist.c b/src/hid/lesstif/netlist.c
index 79e992a..dae93ef 100644
--- a/src/hid/lesstif/netlist.c
+++ b/src/hid/lesstif/netlist.c
@@ -132,7 +132,6 @@ nbcb_select_common (LibraryMenuType *net, int pos, int select_flag)
   ConnectionType conn;
   int i;
 
-  SaveFindFlag (FOUNDFLAG);
   InitConnectionLookup ();
   ResetConnections (true, FOUNDFLAG);
 
@@ -143,7 +142,6 @@ nbcb_select_common (LibraryMenuType *net, int pos, int select_flag)
   SelectConnection (select_flag);
   ResetConnections (false, FOUNDFLAG);
   FreeConnectionLookupMemory ();
-  RestoreFindFlag ();
   IncrementUndoSerialNumber ();
   Draw ();
 }
diff --git a/src/rats.c b/src/rats.c
index 17068a1..9a1c70f 100644
--- a/src/rats.c
+++ b/src/rats.c
@@ -742,7 +742,6 @@ AddAllRats (bool SelectedOnly, void (*funcp) (register ConnectionType *, registe
   changed = false;
   /* initialize finding engine */
   InitConnectionLookup ();
-  SaveFindFlag (DRCFLAG);
   Nets = (NetListType *)calloc (1, sizeof (NetListType));
   /* now we build another netlist (Nets) for each
    * net in Wantlist that shows how it actually looks now,
@@ -778,7 +777,6 @@ AddAllRats (bool SelectedOnly, void (*funcp) (register ConnectionType *, registe
   FreeNetListMemory (Nets);
   free (Nets);
   FreeConnectionLookupMemory ();
-  RestoreFindFlag ();
   if (funcp)
     return (true);
 
@@ -835,7 +833,6 @@ CollectSubnets (bool SelectedOnly)
     }
   /* initialize finding engine */
   InitConnectionLookup ();
-  SaveFindFlag (DRCFLAG);
   /* now we build another netlist (Nets) for each
    * net in Wantlist that shows how it actually looks now,
    * then fill in any missing connections with rat lines.
@@ -868,7 +865,6 @@ CollectSubnets (bool SelectedOnly)
   }
   END_LOOP;
   FreeConnectionLookupMemory ();
-  RestoreFindFlag ();
   return result;
 }
 
diff --git a/src/select.c b/src/select.c
index a5a1045..664f37c 100644
--- a/src/select.c
+++ b/src/select.c
@@ -959,7 +959,6 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
   END_LOOP;
   if (Type & NET_TYPE)
     {
-      SaveFindFlag (FOUNDFLAG);
       InitConnectionLookup ();
       changed = ResetConnections (true, FOUNDFLAG) || changed;
 
@@ -984,7 +983,6 @@ SelectObjectByName (int Type, char *Pattern, bool Flag)
       changed = SelectConnection (Flag) || changed;
       changed = ResetConnections (false, FOUNDFLAG) || changed;
       FreeConnectionLookupMemory ();
-      RestoreFindFlag ();
     }
 
 #if defined(HAVE_REGCOMP)
